extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -220,6 +284,66 @@ export class AmpSelector extends AMP.BaseElement {
   }
 
   /**
+   * Handles click events for the selectables.
+   * @param {!Event} event
+   */
+  clickHandler_(event) {
+    let el = dev().assertElement(event.target);
+    if (!el) {
+      return;
+    }
+    if (!el.hasAttribute('option')) {
+      el = closest(el, e => e.hasAttribute('option'), this.element);
+    }
+    if (el) {
+      this.onOptionSelected_(el);
+    }
+  }
+
+  /**
+   * Handles keyboard events for the selectables.
+   * @param {!Event} event
+   */
+  keyDownHandler_(event) {
+    // Make currently selected option unfocusable
+    this.options_[this.focusedIndex_].tabIndex = -1;
+
+    const dir = this.win.document.body.getAttribute('dir') || 'ltr';
+    let delta = 0;
+
+    switch (event.keyCode) {
+      case 37: // Left
+      case 38: // Up
+        delta = dir === 'rtl' ? 1 : -1;
```

Comment:
only do the inversion for `left` and `right`, `up` and `down` don't change meaning in RTL ( e.g. `down` is `next` and `up` is `previous` in both RTL and LRT)